### DISCUSSION TITLE
Messages with parity error (?) are not deleted

### DISCUSSION THREAD

**Discussion by berrio-23** [opendcs/opendcs#396] [General]
Hello Community

Currently, we have interference in GOES transmissions and some messages have a parity error, code ?; In the opendcs 7.0.8 version, in the decodes module and routing spec, I have changed the parity option with the same result: It does not eliminate the message received with a parity error. Do you have any suggestions? Because despite the thresholds used for the variables, garbage values are passed into the database as a result of interference. It looks like a bug on oppendcs 7.0.8 version. Please, see attachment.
[Messages with parity error.docx](https://github.com/opendcs/opendcs/files/13451587/Messages.with.parity.error.docx)
PD. Opendcs 7.0.6 has the same bug.
Thanks for your comments,
Gustavo

**Comment by MikeNeilson** [comment]
I am unable to open that document. Please put the contents inline in the discussion. 

However, that does sound like a bug; or a bad default config. I would expect the data to still save to the LRGS Archive Database. Is it a routing spec that's erroneously processing the data?

If you are familiar with creating "preformated text in github", You type three back tics, new line ,paste the contents, new line, three more back tics.

**Comment by MikeNeilson** [comment]
Turns out it doesn't appear that the LRGS has ever filtered by parity correctly. I've got a fix ready to merge in for the next release.

**Comment by berrio-23** [comment]
Excellent news for us, Mike.
Thanks a lot
Gustavo
De: Mike Neilson ***@***.***>
Enviado el: martes, 19 de marzo de 2024 6:03 p. m.
Para: opendcs/opendcs ***@***.***>
CC: GUSTAVO DE JESUS MUNOZ BERRIO ***@***.***>; Author ***@***.***>
Asunto: Re: [opendcs/opendcs] Messages with parity error (?) are not deleted (Discussion #396)


¡Cuidado! este correo proviene de un usuario externo, no abras archivos adjuntos ni hagas clic en enlaces sin validar que el remitente y el contenido sean seguros. Nunca entregues tu usuario ni contraseñas a través de enlaces.



Turns out it doesn't appear that the LRGS has ever filtered by parity correctly. I've got a fix ready to merge in for the next release.

-
Reply to this email directly, view it on GitHub<https://github.com/opendcs/opendcs/discussions/396#discussioncomment-8846800>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BEFI3R7VJCBAJXDN6JRU4SDYZC72LAVCNFSM6AAAAAA7X6HYP6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DQNBWHAYDA>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

El contenido de este documento y/o sus anexos son para uso exclusivo de su destinatario intencional y puede contener Información legalmente protegida por ser privilegiada o confidencial. Si usted no es el destinatario intencional de este documento por favor Infórmenos de inmediato y elimine el documento y sus anexos. Igualmente, el uso indebido, revisión no autorizada, retención, distribución, divulgación, reenvío, copia, impresión o reproducción de este documento y/o sus anexos está estrictamente prohibido y sancionado legalmente. Agradecemos su atención. Grupo Empresarial EPM.

The contents of this document and/or its attachments are for exclusive use of the intended recipient and may contain privileged or confidential information. If you are not the intended recipient of this document, please immediately reply to the sender and delete this information and its attachments from your system. Likewise, the misuse, unauthorized review, any retention, dissemination, distribution, disclosure, forwarding, copying, printing or reproduction of this transmission, including any attachments, is strictly prohibited and punishable by law. Thank you for your attention. Grupo Empresarial EPM.
